Sources/TimeFreeze.swift

```diff
@@ -10,7 +10,7 @@ struct TimeFreeze {
     private let offset: TimeInterval
 
     /// The stable timestamp adjusted by the most acurrate offset known so far.
-    var adjustedTimestamp: TimeInterval? {
+    var adjustedTimestamp: TimeInterval {
```

Comment:
surprised this doesn't cause build failures, shouldn't making this non-optional also require us to remove some optional chaining somewhere?

Reply:
It's just a coincidence this didn't break anything. Is because we are using it on an optional chaining such as `self.whatever?.adjustedTimestamp` that as a whole returns an optional regardless